src/t8_cmesh/t8_cmesh_examples.c

```diff
@@ -1078,36 +1088,42 @@ t8_cmesh_set_vertices_3D (t8_cmesh_t cmesh, const t8_eclass_t eclass, const doub
   for (t8_locidx_t hex_z_id = 0; hex_z_id < hexs_z; hex_z_id++) {
     for (t8_locidx_t hex_y_id = 0; hex_y_id < hexs_y; hex_y_id++) {
       for (t8_locidx_t hex_x_id = 0; hex_x_id < hexs_x; hex_x_id++) {
+        if (use_axis_aligned_geom && eclass == T8_ECLASS_HEX) {
```

Comment:
Should we abort if `use_axis_aligned_geom && eclass != T8_ECLASS_HEX`?

Reply:
Resturctured the program a little and added this feature too.